docs/gboard-db-notes.md

```diff
@@ -0,0 +1,69 @@
+# Database names and content
+
+Bellow are all the discovered contents existent in some tables.
+Results will be shown by database followed by table.
+Each table will be described using the following syntax:
+ | | | |
+ | :--- | :--- | :---: |
+ | status | whether the table is under investigation or fully investigated | ON GOING, FINISHED |
+ | content | what information resides in the table | |
+ | certainty | whether the content and column descriptions are 100% accurate | YES, NO |
+
+## trainingcache2.db (9.4.11.312687073)
+
+This database is used to store inputs from GBoard. It detects individual key presses, deleted words, swiped words.
+
+### SQLITE_SEQUENCE
+ * **status:** ON GOING
+ * **content:** Gathers all tables which were updated at least once.
+ * **certainty:** NO
+
+#### columns
+ * **NAME:** table's name 
+ * **SEQ:** amount of content in the table
+
+
+### D_TABLE
+ * **status:** ON GOING
+ * **content:** Deleted words
+ * **certainty:** YES
+
+#### columns
+ * **_ID:** internal id
+ * **_TIMESTAMP:** time elapsed since the Unix epoch (ms)
+ * **_PAYLOAD:** ?
+ * **F1:** ?
+ * **F2:** global typed sequence
+ * **F3:** ?
+ * **F4:** word size
+ * **F5:** word
+ * **F6:** ?
+
+
+### TF_TABLE
+ * **status:** ON GOING
+ * **content:** Typed words
+ * **certainty:** YES
+
+#### columns
+ * **_ID:** internal id
+ * **_TIMESTAMP:** time elapsed since the Unix epoch (ms)
+ * **_PAYLOAD:** ?
+ * **F1:** ?
+ * **F2:** global typed sequence
+ * **F3:** word
```

Comment:
this can be confusing. It can be a single character as well. I would rename this to "character sequence" or "character/word"

Reply:
Yup I was defining a word as a sequence which is not the best approach. Changed it to string as it can contain emojis as well.